bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RefreshingOAuth2CredentialsInterceptor.java

```diff
@@ -94,13 +89,14 @@
      * This specifies how far in advance of a header expiration do we consider the token stale. The
      * Stale state indicates that the interceptor needs to do an asynchronous refresh.
      */
-    static final int TOKEN_STALENESS_MS = 75 * 1000;
+    static final int TOKEN_STALENESS_MS = 6 * 60 * 1000; // 6 minutes
```

Comment:
nit: TimeUnit.MINUTEs.toMillis(6) should be more readable.  The same nit applies for for TOKEN_EXPIRES_MS